scalability_and_performance/ztp-zero-touch-provisioning.adoc

```diff
@@ -1,21 +1,20 @@
 // CNF-1500 assembly
 [id="ztp-zero-touch-provisioning"]
-= Zero touch provisioning
+= Deploying distributed units at scale in a disconnected environment
```

Comment:
If you're retitling this assembly, you might want to also rename the file, particularly if it hasn't been published yet.

Reply:
Just made this change...